draft-ietf-webpush-protocol.xml

```diff
@@ -453,9 +453,15 @@ Location: https://push.example.net/d/qDIYHNcfAIPP_5ITvURr-d6BGtYnTRnk
 ]]></artwork>
       </figure>
       <t>
-        A push service MAY generate a 413 (Payload Too Large) status code in
-        response to requests that include an entity body that is too large.
-        Push services MUST NOT generate a 413 status code in responses to an
+        A push service MAY return a 429 (Too Many Requests) status code <xref target="RFC6585"/>
+        when an application server has exceeded its rate limit for push message
+        delivery. The push service SHOULD also include a Retry-After header <xref target="RFC7231"/>
+        to indicate how long the application server must wait before it makes another request. 

```

Comment:
Nah, the application server would be left guessing.  After all, there are valid reasons to send 429 on any scope.

The best response to this is for the application server to stop pushing to *this* resource for the requested period, but not stop for other resources until they responded.  Then you might have some heuristic to decide that all endpoints are busy so that you can stop entirely.